src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -341,7 +341,7 @@ class Activity extends React.Component {
   }
 }
 
-const OrganizationDashboard = createReactClass({
+export const OrganizationDashboard = createReactClass({
```

Comment:
Let's leave this as a default export to be consistent with our other views. I think we should use named exports for components that are only used for tests (like `ProjectList`)